data/autoyast_sle12/autoyast_wicked_x86_64.xml

```diff
@@ -16,6 +16,15 @@
       <append> splash=verbose </append>
     </global>
   </bootloader>
+  <add-on>
+    <add_on_products config:type="list">
+      <listentry>
+        <media_url><![CDATA[http://download.suse.de/ibs/SUSE:/SLE-12-SP5:/GA:/TEST/images/repo/SLE-12-SP5-SDK-POOL-x86_64-Media1/]]></media_url>

```

Comment:
Does it have to be fixed version ? I'm just wondering if QAM would like to reuse this test for older SPs.
Or for that case new file will have to be created?

Reply:
for "historical reasons" there is no easy way to insert "12-SP5" here. if you will use {{VERSION}} variable you will get "12.5" instead . So I am afraid that if QAM **would** like to use it they **would** need to invest there own time to fix it. Too much uncertainty for too invest time into proper fix here .

Reply:
ok, fine,